seqr/views/apis/individual_api.py

```diff
@@ -118,42 +114,101 @@ def edit_individuals_handler(request, project_guid):
 
     request_json = json.loads(request.body)
 
-    if 'form' not in request_json:
+    modified_individuals_list = request_json.get('individuals')
+    if modified_individuals_list is None:
         return create_json_response(
-            {}, status=400, reason="Invalid request: 'form' key not specified")
-
-    form_data = request_json['form']
+            {}, status=400, reason="'individuals' not specified")
 
-    modified_individuals_by_guid = form_data.get('modifiedIndividuals')
-    if modified_individuals_by_guid is None:
-        return create_json_response(
-            {}, status=400, reason="'modifiedIndividuals' not specified")
+    update_individuals = {
+        ind['individualGuid']: (Individual.objects.get(guid=ind['individualGuid']), ind) for ind in modified_individuals_list
+    }
 
-    # edit individuals
-    modified_individuals_list = list(modified_individuals_by_guid.values())
+    errors = []
```

Comment:
what's the reason for switching away from `validate_fam_file_records` here?

Reply:
It assumes that this is the first upload and that all individuals you care about are in the update data. So for example, if you modify an individual and not their child `validate_fam_file_records` would not check if the sex/famly for that individual were still valid because it would not know that the unmodified child exists.